crm_timesheet/__manifest__.py

```diff
@@ -3,13 +3,14 @@
 # Copyright 2017 Tecnativa - David Vidal
 # Copyright 2019 Tecnativa - Jairo Llopis
 # Copyright 2020 Tecnativa - Pedro M. Baeza
+# Copyright 2023 Tecnativa - Carolina Fernandez
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     "name": "CRM Timesheet",
     "category": "Customer Relationship Management",
-    "version": "15.0.1.0.0",
-    "depends": ["crm", "project_timesheet_time_control"],
+    "version": "16.0.1.0.0",
+    "depends": ["crm", "project_timesheet_time_control", "hr_timesheet_task_stage"],
```

Comment:
Why do you need `hr_timesheet_task_stage`?

Reply:
There is an inherit view in views/hr_timesheet_view.xml refering to this module: hr_timesheet_task_stage